### PR TITLE
[CI] List this repo's name in workflow

### LINKED ISSUE #2: [CI] Slack Notifier: send message to #buzz channel on star
#### Current Behavior
This newly minted repo needs the two placeholders for repo name updated in the https://github.com/layer5io/mesheryctl-aliases/blob/master/.github/workflows/slack.yml workflow.

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by josephmidura**
Signed-off-by: Joseph Midura <josephmidura632@gmail.com>

**Description**

This PR fixes #2 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/slack.yml [hunk 1]
```diff
@@ -7,10 +7,10 @@ jobs:
...
-        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/meshery' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
+        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/mesheryctl-aliases' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
     - name: Notify slack
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
       uses: pullreminders/slack-action@master
       with:
-        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred <new-repo>! (https://github.com/layer5io/meshery/stargazers) Total ⭐️: ${{env.STARS}}\"}'
\ No newline at end of file
+        args: '{\"channel\":\"CSK7N9TGX\",\"text\":\"Someone just starred <new-repo>! (https://github.com/layer5io/mesheryctl-aliases/stargazers) Total ⭐️: ${{env.STARS}}\"}'
```
Comment (shuklaritvik06) [inline .github/workflows/slack.yml]: Hey!, Thank you for the PR , please write this repo name in the place of `<new-repo>`

**Review by shuklaritvik06 — CHANGES_REQUESTED** [review]

File: .github/workflows/slack.yml (hunk 1 above)
Comment (leecalcote) [reply .github/workflows/slack.yml]: Great feedback.

**Comment by josephmidura** [comment]
Thanks for the feedback. Change made.

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :)

**Review by leecalcote — APPROVED** [review]
Excellent.

**Pull request merged**